views/dashboard.tmpl

```diff
@@ -9,8 +9,9 @@
          </div>
         <div class="tyk-col--lg-9 tyk-col card">
           <h2 class="card__header">Hello
-            {{if .CurrentUser}}
-              {{.CurrentUser.DisplayName}}</h5>
+          {{ $user := CurrentUser req }}
+            {{if $user}}
```

Comment:
Nice ! Can't we assume that there will be a `{{ $user }}`  ? Why would that fail ?

Reply:
Up to you to simplify the logic here